pocs/camera/camera.py

```diff
@@ -1,13 +1,30 @@
 from .. import PanBase
-from ..focuser.focuser import AbstractFocuser
+
 from ..utils import error
 from ..utils import listify
 from ..utils import load_module
+from ..utils import images
+from ..utils import current_time
+
+from ..focuser.focuser import AbstractFocuser
+
+from astropy.io import fits
+from astropy.stats import sigma_clipped_stats
+from astropy.modeling import models, fitting
+
+import numpy as np
+
+import matplotlib
```

Comment:
As discussed, let's move `autofocus` and all plot-related dependencies into the `Focuser` itself. Can make plotting smart or not within that class but might not need it.